README.md

```diff
@@ -19,5 +19,22 @@ You can use the provided macro in lib.sls to generate the required configuration
 ### Usage with Zend Server module
 The lookup part (commented in pillar.example) is **required** when using non-default PHP stacks such as Zend Server.
 
+## PHP-FPM Versions
+
+The phpfpm module recognizes either the vanilla PHP-FPM 5.5 installation, or any combination of PHP-FPM 5.6 and PHP-FPM 7, provided by Ondřej Surý via the PPA.
```

Comment:
Klopt niet helemaal want hij defaults naar de "OS version". 
Dat is 5.5 voor 14.04 maar voor 16.04 is dit 7.0. 

Wellicht nog raadzaam daar een remark van in te zetten dat die Sury repo dan natuurlijk overbodig is :)